Packs/Intezer/ReleaseNotes/1_4_0.md

```diff
@@ -0,0 +1,18 @@
+
+#### Integrations
+##### Intezer v2
+- Send the original file name in analyze-by-file command
```

Comment:
How does it impact the users? As I see it, this happens "behind the scenes", and might confuse users.

Reply:
I can remove it, but the file name is presented in Intezer Analyze, which currently it isn't informative (it usually something like that `ad0 @ 821`.